src/raiden_libs/service_registry.py

```diff
@@ -54,6 +111,21 @@ def cli() -> None:
 
 @blockchain_options(contracts=[CONTRACT_SERVICE_REGISTRY])
 @cli.command()
+@click.option(
+    "--accept-disclaimer",
+    type=bool,
+    default=False,
+    help="Bypass the experimental software disclaimer prompt",
+    is_flag=True,
+)
+# TODO: hide this from help
+@click.option(
+    "--accept-all",
+    type=bool,
+    default=False,
+    help="Bypass all questions (Not recommended)",
+    is_flag=True,
```

Comment:
The flag you are looking for is `hidden`.
```suggestion
@click.option(
    "--accept-all",
    type=bool,
    default=False,
    help="Bypass all questions (Not recommended)",
    is_flag=True,
    hidden=True,
```

Reply:
Thanks, I didn't find that in the help with a quick look.